libnavigation-ui/src/main/java/com/mapbox/navigation/ui/map/NavigationMapboxMap.java

```diff
@@ -1327,6 +1334,16 @@ public Builder useSpecializedLocationLayer(boolean useSpecializedLocationLayer)
       return this;
     }
 
+    /**
+     * @param padding indicates the size of the bounding box used when clicking the map to select
+     * alternative routes. The padding value is added to the the sides, top and bottom of the click
+     * click point.
+     */
+    public Builder withRouteClickPadding(int padding) {
+      this.routeClickPadding = padding;
+      return this;
+    }
+
```

Comment:
This should have the same extensive doc as `NavigationMapRoute`'s buidler method, and all of the comments about DP/raw pixels and docs adjustments apply here as well.